webcompat/views.py

```diff
@@ -349,7 +343,10 @@ def show_issue(number):
     if session.get('show_thanks'):
         flash(number, 'thanks')
         session.pop('show_thanks')
-    return render_template('issue.html', number=number)
+    issue_data = proxy_issue(number)
+    return render_template('issue.html',
+                           issue_data=json.loads(issue_data[0]),
+                           json_data=issue_data[0].decode('utf-8'))
```

Comment:
There are two methods to convert data-types :) 
https://github.com/webcompat/webcompat.com/blob/ccedceb3d38c07b9a6611b8ebed57553ac1b76e0/webcompat/helpers.py#L702-L717

```suggestion
                           json_data=helpers.to_str(issue_data[0])
```

Reply:
Oh! I didn't notice these. Perfect, thanks.